tiller_python_service/controller.py

```diff
@@ -12,6 +12,29 @@
 
 LOGGER = logging.getLogger(__name__)
 
+def my_new_sort(arr):
```

Comment:
This sort function is very inefficient, can you make it a merge sort instead? help tiller

Reply:
This sort function is very inefficient, can you replace it with a merge sort instead? help tiller